docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -1,78 +1,84 @@
 ---
 title: Database Access GUI Clients
-description: How to configure graphical database clients
+description: How to configure graphical database clients for Teleport Database Access.
```

Comment:
Does MongoDB setup work this https://www.mongodb.com/try/download/compass?

Reply:
Kind of. I was able to connect but found out that it doesn't work well with our RBAC - i.e. if your Teleport role prohibits access to specific database, for some reason Compass refuses to load anything. It works otherwise - I'll see if we can maybe return a more friendly access denied error to make it work.

I also tried some other client called Robo 3T (https://robomongo.org/) - this one works fine. Not sure if it's popular or not though.

Reply:
That's good to know. No need to add anything now, but it'll be interesting to see what the community requests.